### PR TITLE
Merge branch-UG-Qiaoran with Master

### LINKED ISSUE #7: Update add and edit gender feature of UserGuide.md

### PR DESCRIPTION
**Pull request by Qiaoran-M**

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -72,43 +72,73 @@ Shows a message explaning how to access the help page.
...
 
-### Adding a person: `add`
+Adds a contact to the address book.
 
-Adds a person to the address book.
+Format: `add n/NAME p/PHONE_NUMBER e/EMAIL a/ADDRESS [g/GENDER] [d/DOB] [ph/NEW_PURCHASE_DETAIL] [t/TAG]`
 
-Format: `add n/NAME p/PHONE_NUMBER e/EMAIL a/ADDRESS [t/TAG]…​`
+- The compulsory parameters are: name (n), phone number (p), email (e), address (a)
+- The parameters in [ ] are optional parameters, including gender (g), date of birth (d), purchase history (ph), tag (t).
```
Comment (Benjamin-Sim) [inline docs/UserGuide.md]: Perhaps format the command prefixes as inline code to signal to the reader that they are part of the command?
i.e. name (`n`), phone number (`p`), email (`e`), address (`a`)

File: docs/UserGuide.md [hunk 2]
```diff
@@ -72,43 +72,73 @@ Shows a message explaning how to access the help page.
...
+* `add n/John Doe p/98765432 e/johnd@example.com a/John street, block 123, #01-01 g/m d/20 MAR 2002
+  ph/[12.04.2022, 101.25, 3, Orange, Umbrella, Soap]`
+* `add n/Betsy Crowe e/betsycrowe@example.com a/Newgate Prison p/1234567 t/friend t/criminal g/f g/14.12.1998
+  ph/[13.05.2021 12.25, 3, Sweets, Banana, Battery]`
+
+__Optional Parameter 1: Gender__
+
+Format: `add n/NAME p/PHONE_NUMBER e/EMAIL a/ADDRESS [g/GENDER]`
+
+- Adds gender to a person in the contact list. The genders accepted by the contact list are: M/m for male,  F/f for female.
```
Comment (Benjamin-Sim) [inline docs/UserGuide.md]: Same as above.
i.e. `M`/`m` for male,  `F`/`f` for female.

File: docs/UserGuide.md [hunk 3]
```diff
@@ -72,43 +72,73 @@ Shows a message explaning how to access the help page.
...
 *  `edit 2 n/Betsy Crower t/` Edits the name of the 2nd person to be `Betsy Crower` and clears all existing tags.
+*  `edit 3 n/Charlotte g/F d/3.4.1998 ph/3,[20.03.2022, 101.25, Chair, Table, Banana]` Edits the 3rd person’s contact:
+   edits name to be `Charlotte`, edits gender to be `Female`, edits date of birth to be `3.4.1998`,
+   edits the purchase history to be `3,[20.03.2022, 101.25, Chair, Table, Banana]`.
+
+__Optional Parameter 1: Gender__
+
+Format: `edit INDEX [g/GENDER]`
+
+- Edits the gender of a person in the contact list. The genders accepted by the contact list are: M/m for male,  F/f for female.
```
Comment (Benjamin-Sim) [inline docs/UserGuide.md]: Same as above.

File: docs/UserGuide.md [hunk 4]
```diff
@@ -72,43 +72,73 @@ Shows a message explaning how to access the help page.
...
+- INDEX must be positive integers (i.e 1,2,3…)
+- INDEX must be within the range of the contact list index (i.e from 1 to size of contact list).
```
Comment (Benjamin-Sim) [inline docs/UserGuide.md]: Perhaps format INDEX as inline code as well?

**Review by Benjamin-Sim — COMMENTED** [review]
All good except for those minor formatting issues 👍

File: docs/UserGuide.md (hunk 4 above)
Comment (Qiaoran-M) [reply docs/UserGuide.md]: Okk I will change this, thanks for the review!

**Review by Benjamin-Sim — APPROVED** [review]
LGTM! 👍

**Pull request merged**